src/test/rgw/restler/openapi.yaml

```diff
@@ -34,116 +34,8 @@ externalDocs:
-          - us-west-2
-          - us-gov-west-1
-          - eu-west-1
-          - ap-northeast-1
-          - ap-southeast-1
-          - ap-southeast-2
-          - sa-east-1
-        default: us-east-1
-      dash-or-dot:
-        description: The service/region URL separator
-        enum:
-          - .
-          - '-'
-        default: .
-    description: The Amazon S3 multi-region endpoint
-  - url: 'https://s3{dash-or-dot}{region}.amazonaws.com'
-    variables:
-      region:
-        description: The AWS region
-        enum:
-          - us-east-1
-          - us-west-1
-          - us-west-2
-          - us-gov-west-1
-          - eu-west-1
-          - ap-northeast-1
-          - ap-southeast-1
-          - ap-southeast-2
-          - sa-east-1
-        default: us-east-1
-      dash-or-dot:
-        description: The service/region URL separator
-        enum:
-          - .
-          - '-'
-        default: .
-    description: The Amazon S3 multi-region endpoint
-  - url: 'http://s3.amazonaws.com'
-    variables: {}
-    description: The general Amazon S3 endpoint for US East (N. Virginia)
-  - url: 'https://s3.amazonaws.com'
-    variables: {}
-    description: The general Amazon S3 endpoint for US East (N. Virginia)
-  - url: 'http://s3.{region}.amazonaws.com'
-    variables:
-      region:
-        description: The AWS region
-        enum:
-          - us-east-2
-          - us-gov-east-1
-          - ca-central-1
-          - eu-north-1
-          - eu-west-2
-          - eu-west-3
-          - eu-central-1
-          - eu-south-1
-          - af-south-1
-          - ap-northeast-2
-          - ap-northeast-3
-          - ap-east-1
-          - ap-south-1
-          - me-south-1
-        default: us-east-2
-    description: The Amazon S3 multi-region endpoint
-  - url: 'https://s3.{region}.amazonaws.com'
-    variables:
-      region:
-        description: The AWS region
-        enum:
-          - us-east-2
-          - us-gov-east-1
-          - ca-central-1
-          - eu-north-1
-          - eu-west-2
-          - eu-west-3
-          - eu-central-1
-          - eu-south-1
-          - af-south-1
-          - ap-northeast-2
-          - ap-northeast-3
-          - ap-east-1
-          - ap-south-1
-          - me-south-1
-        default: us-east-2
-    description: The Amazon S3 multi-region endpoint
-  - url: 'http://s3.{region}.amazonaws.com.cn'
-    variables:
-      region:
-        description: The AWS region
-        enum:
-          - cn-north-1
-          - cn-northwest-1
-        default: cn-north-1
-    description: The Amazon S3 endpoint for China (Beijing) and China (Ningxia)
-  - url: 'https://s3.{region}.amazonaws.com.cn'
-    variables:
-      region:
-        description: The AWS region
-        enum:
-          - cn-north-1
-          - cn-northwest-1
-        default: cn-north-1
-    description: The Amazon S3 endpoint for China (Beijing) and China (Ningxia)
+  - url: 'http://localhost:8000'
```

Comment:
this would work only with vstart. which is good (for now)
later on we would need to be able to easily replace that in teuthology